environments/prod/backend_lb_config_arm_to_tf.yaml

```diff
@@ -9,3 +9,10 @@ gateways:
       # CNP
       - product: plum
         component: recipe-backend
+
```

Comment:
As no applications added into this second AppGw it was failing plans on the pipeline. SO have commented out until needed.